src/J2N/Runtime/InteropServices/CollectionMarshal.cs

```diff
@@ -102,7 +102,7 @@ public static void SetCount<T>(List<T> list, int count)
 
             if (count > list.Capacity)
             {
-                list.Grow(count);
+                list.EnsureCapacity(count);
```

Comment:
`Grow()` can be called here because we are already checking that we have exceeded capacity. This is a bit more efficient because there are no bounds checks.

https://github.com/dotnet/runtime/blob/v9.0.4/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/CollectionsMarshal.cs#L128